### DISCUSSION TITLE
No updated pose for CF cf1

### DISCUSSION THREAD

**Discussion by cfc-ray** [USC-ACTLab/crazyswarm#736] [General]
Hello,

I am trying to run the 'hello_world' script to have just one drone hover, but I am having an issue when I start up a crazyswarm server. Everything appears to boot up right, and by that I mean crazyswarm is able to connect to the crazyflie and change some parameters. It is also able to connect to VICON and start up Rviz just fine. However, once Rviz is opened, I get an error saying that there is 'no updated pose for CF cf1'. Here is the output from my terminal:

![Screenshot from 2023-04-24 12-33-42](https://user-images.githubusercontent.com/83977232/234073333-7a0f5341-8c17-433c-8aab-cab5137c6041.png)
![Screenshot from 2023-04-24 12-33-46](https://user-images.githubusercontent.com/83977232/234073376-36a8dc63-29c9-487a-a4d1-df662e4d804f.png)
![Screenshot from 2023-04-24 12-33-49](https://user-images.githubusercontent.com/83977232/234073378-c54677ef-287f-43a8-b502-0be9669a5801.png)
![Screenshot from 2023-04-24 12-33-56](https://user-images.githubusercontent.com/83977232/234073379-31a2b43f-2448-437b-8820-64759a3237d7.png)
![Screenshot from 2023-04-24 12-34-03](https://user-images.githubusercontent.com/83977232/234073380-3bdb8f08-ba8d-4fd3-81c3-78e1e6d49cb9.png)
![Screenshot from 2023-04-24 12-34-08](https://user-images.githubusercontent.com/83977232/234073382-5e67e5da-9f68-40d7-9fc7-379aef42415c.png)
![Screenshot from 2023-04-24 12-34-11](https://user-images.githubusercontent.com/83977232/234073383-49bb1dfe-a84d-4cff-87db-f1e336c85b06.png)
![Screenshot from 2023-04-24 12-34-13](https://user-images.githubusercontent.com/83977232/234073384-4a06fb39-df42-40d2-bbd8-024469735eee.png)

I tried running 'mocap_helper.launch', and this is my output:
![Screenshot from 2023-04-24 12-40-28](https://user-images.githubusercontent.com/83977232/234074391-b02e6754-626a-41c4-8459-41171c1e5614.png)

So it seems like crazyswarm is able to read the position and orientation of the crazyflie, but for some reason it's not able to update the crazyflie's pose.

I have received this same error in the past, and it always arose if I forgot to track the crazyflie through VICON, or if I named it something other than 'cf1'. In this case though I am doing both of those things, so I'm not sure what else could be happening to cause this issue.

Thanks in advance if anybody has run into any similar issues and is willing to help me out!

**Comment by CH4JW614** [comment]
Hello,  have you solved this problem？If you solved，can you help me
